### ISSUE TITLE
Proposal: Stop using sphinx-rtd-theme

### ISSUE THREAD

**Issue by auscompgeek** [robotpy/robotpy-docs#53]
The latest version of sphinx-rtd-theme (1.1.1) is incompatible with Sphinx 6: the navigation sidebar depends on jQuery. https://blog.readthedocs.com/sphinx6-upgrade/

Maintenance on sphinx-rtd-theme appears to be quite slow: there was over a year gap between 1.0 and 1.1.

The wider Python community seem to have moved to other Sphinx themes. In particular, [furo](https://github.com/pradyunsg/furo), by one of the PyPA folks, appears to have become quite popular, and has dark theme support.

**Comment by virtuald** [comment]
Meh.

**Comment by TheTripleV** [comment]
I don't think this matters as much as I expect most docs to be migrates over anyway.

**Comment by auscompgeek** [comment]
We'd still have API docs though?
